task/06-conditions-n-loops-tasks.js

```diff
@@ -270,9 +310,13 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
-}
+    let res = num.toString().split('').map(el=>+el).reduce((acc,el)=>acc+=el,0)
 
+    if(res > 9){
+      res = res.toString().split('').map(el=>+el).reduce((acc,el)=>acc+=el,0)
```

Comment:
It will not work when res have more than 9 digits after second iteration.
So universal solotion should have some loop with condition like `while count of digits is  more than 9`